src/test/scala/org/broadinstitute/dsde/vault/services/analysis/AnalysisDescribeServiceSpec.scala

```diff
@@ -69,6 +92,17 @@ class AnalysisDescribeServiceSpec extends VaultFreeSpec with AnalysisDescribeSer
               files.get isDefinedAt "bam"
               files.get isDefinedAt "bai"
               files.get isDefinedAt "vcf"
+
+              version match {
+                case Some(x) if x > 1 =>
+                  analysisResponse.properties shouldNot be(empty)
```

Comment:
I like the checks at lines 44-49 better.  Is there a reason these are different?  If not, duplicate the checks from above.